modules/tensor_mechanics/tests/2D_geometries/2D-RZ_finiteStrain_resid.i

```diff
@@ -34,7 +34,7 @@
 [Modules/TensorMechanics/Master]
   [./all]
     strain = FINITE
-    add_variables = true
+    add_displacement_variables = true
     save_in = 'force_r force_z'
```

Comment:
Not necessary for this commit, but it would be nice to add an option for it to set up the AuxVariables and everything related to saving the residual to get the reactions with an option to this Action.